src/main/scala/ch/epfl/bluebrain/nexus/iam/types/Caller.scala

```diff
@@ -0,0 +1,4 @@
+package ch.epfl.bluebrain.nexus.iam.types
+import ch.epfl.bluebrain.nexus.iam.types.Identity.Subject
+
+final case class Caller(subject: Subject, identities: Set[Identity])
```

Comment:
docs? it's also important to note if the subject is also included in the identity set.